src/EventStore.Core/Services/VNode/ClusterVNodeController.cs

```diff
@@ -579,11 +580,11 @@ public class ClusterVNodeController<TStreamId> : ClusterVNodeController, IHandle
 		}
 
 		private void Handle(AuthenticationMessage.AuthenticationProviderInitialized message) {
-			_startSubsystems();
+			_startSubsystems().Wait();
```

Comment:
maybe this can be a continuewith and the continuation do the remainder of the method (if no exception)